install_config/configuring_local.adoc

```diff
@@ -208,3 +208,227 @@ $ chcon -R unconfined_u:object_r:svirt_sandbox_file_t:s0 /mnt/local-storage/
 ====
 Omitting any of these steps may result in the wrong PV being created.
 ====
+
+[[local-volume-raw-block-devices]]
+=== Raw Block Devices
+It is possible to statically provision also raw block devices using the local
+volume provisioner. This feature is disabled by default and requires additional
+configuration.
+
+. Enable the `BlockVolume` feature gate on all masters and nodes.
+Edit or create the master configuration file on all masters
+(*_/etc/origin/master/master-config.yaml_* by default) and add `BlockVolume=true`
+under the `apiServerArguments` and `controllerArguments` sections:
++
+[source, yaml]
+----
+apiServerArguments:
+   feature-gates:
+   - BlockVolumes=true
+...
+
+ controllerArguments:
+   feature-gates:
+   - BlockVolumes=true
+...
+----
+
+. On all nodes, edit or create the node configuration file
+(*_/etc/origin/node/node-config.yaml by default_*) and add `BlockVolumes=true`
+feature gate under `kubeletArguments`:
++
+[source, yaml]
+----
+kubeletArguments:
+   feature-gates:
+   - BlockVolumes=true
+----
+
+. Restart all the nodes and master.
+
+[[local-volume-prepare-block-devices]]
+==== Prepare the Block Devices
+Before starting the provisioner all the block devices that should be available
+to the pods need to be linked to the *_/mnt/local-storage/<storage class>_* directory
+structure. Example: to make a device *_/dev/dm-36_* available:
+
+. Create a directory for its `StorageClass` in *_/mnt/local-storage_*:
++
+[source, bash]
+----
+$ mkdir -p /mnt/local-storage/block-devices
+----
+
+. Create a symbolic link that would point to the device:
++
+[source, bash]
+----
+$ ln -s dm-uuid-LVM-1234 /dev/dm-36
```

Comment:
ln -s /dev/dm-36 /mnt/local-storage/block-devices/dm-uuid-LVM-1234 ?

Reply:
Yes. This is wrong.